# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/PHttpConnectionMgr.ipdl
# Commit: 2502449c2e24
# Full Hash: 2502449c2e24993d495c886c717e7f9546cd9e17
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-05 21:44:54
# Regressor Bug: 1497249
# File Overlap Count: 4
# Description:
#   Bug 1497249 - P2: ipdl for nsIWebSocketConnection r=michal
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30624
# ==============================================================================

diff -r 432c7e9dc19f -r 2502449c2e24 netwerk/protocol/http/PHttpConnectionMgr.ipdl
--- a/netwerk/protocol/http/PHttpConnectionMgr.ipdl	Mon Aug 03 10:03:08 2020 +0000
+++ b/netwerk/protocol/http/PHttpConnectionMgr.ipdl	Wed Aug 05 15:17:38 2020 +0000
@@ -8,6 +8,7 @@
 include protocol PAltSvcTransaction;
 include protocol PSocketProcess;
 include protocol PHttpTransaction;
+include protocol PWebSocketConnection;
 
 include NeckoChannelParams;
 
@@ -17,6 +18,7 @@
 async refcounted protocol PHttpConnectionMgr
 {
   manager PSocketProcess;
+  manages PWebSocketConnection;
 
 child:
   async __delete__();
@@ -33,6 +35,7 @@
   async SpeculativeConnect(HttpConnectionInfoCloneArgs aConnInfo,
                            SpeculativeConnectionOverriderArgs? aOverriderArgs,
                            uint32_t aCaps, PAltSvcTransaction? aTrans);
+  async PWebSocketConnection(PHttpTransaction aTransWithStickyConn);
 };
 
 } // namespace net